worker/src/main/java/org/apache/celeborn/service/deploy/worker/storage/FileWriter.java

```diff
@@ -105,7 +103,17 @@ public FileWriter(
     if (!fileInfo.isHdfs()) {
       channel = new FileOutputStream(fileInfo.getFilePath()).getChannel();
     } else {
-      stream = StorageManager.hdfsFs().create(fileInfo.getHdfsPath(), true);
+      try {
+        StorageManager.hdfsFs().create(fileInfo.getHdfsPath(), true).close();
```

Comment:
`hdfsFs` is a little redundant name, suggest to rename it to `hadoopFs`, in all place, the reason is:

- we may support other hadoop compatible file system throw hadoop FileSystem API in the future, e.g. s3 oss alluxio
- in hdfsFs, "fs" is redundant